src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,78 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import mate.academy.lib.Inject;
+
+@Entity
+@Inject
+@Table(name = "movie sessions")

```

Comment:
The name of the table should not contain spaces as it may lead to errors. Rename the table to `movie_sessions`.